packages/features/portfolio/src/hooks/use-vault-holdings-table.tsx

```diff
@@ -224,6 +235,20 @@ export const useVaultHoldingsTable = () => {
       const totalPoints =
         arbPoints?.find(({ token }) => token === v.vaultShares.tokenId)
           ?.points || 0;
+      const vaultReinvestmentData = reinvestmentData?.[v.vaultAddress];
```

Comment:
Do a check in case the vault address is not in the object or else this will crash